src/System.Linq/src/System.Linq.csproj

```diff
@@ -24,7 +24,6 @@
     <Compile Include="..\..\Common\src\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
We're on the "nothing" plan on github (or at least for the libraries already up there, this is the only one). So it sounds like that's not the case for the ones we're still building internally?